bigframes/session/metrics.py

```diff
@@ -64,13 +66,12 @@ def get_performance_stats(
         # dry run stats are just predictions of the real run
         slot_millis = 0
 
-    exec_seconds = (
-        (query_job.ended - query_job.created).total_seconds()
-        if query_job.created is not None and query_job.ended is not None
-        else None
-    )
+    if query_job.created is not None and query_job.ended is not None:
+        execution_secs = (query_job.ended - query_job.created).total_seconds()
+    else:
+        return None
```

Comment:
there are multiple `return None` statements which makes it a bit hard to follow. Can we do it only once in the beginning `if query_job.state != "DONE"`, otherwise always return a legit tuple?

Reply:
Seems query_job.state != "DONE" does not guarantee the metrics exist for case like mocks and dry_run, so maybe this is more reliable.